Assets/Scripts/Game Controllers/WinConditionController.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class WinConditionController : MonoBehaviour
+{
+    private void OnTriggerEnter2D(Collider2D collision)
+    {   
+        Debug.Log(collision.name);
+        if(collision.name == "Player")
+        {
+            // Loads pause scene
+            SceneManager.LoadScene(0);
```

Comment:
This should use the scene name as a string instead of the scene build index so that it's clear which scene is being loaded. Then, the comment can be removed.

Reply:
- [ ] Update